src/v2i-hub/PedestrianPlugin/src/FLIRWebSockAsyncClnSession.cpp

```diff
@@ -6,10 +6,44 @@ using namespace boost::property_tree;
 
 namespace PedestrianPlugin
 {
+    int 
+    FLIRWebSockAsyncClnSession::calculateMinuteOfYear(int year, int month, int day, int hour, int minute, int second) {
+    // Set up current time
+    std::tm currentTime = {};
+    currentTime.tm_year = year - 1900;
```

Comment:
Provide brief comments for this calculation for year and month

Reply:
Added comments